app/pings.yaml

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$schema: moz://mozilla.org/schemas/glean/pings/1-0-0
+
+session_end:
+  description: >
+    These pings are sent at the end of a session. (when Firefox Reality switches to the background.)
+    We usually send search and UI control metrics at the end of a session.
+  include_client_id: false
+  bugs:
+    - https://github.com/MozillaReality/FirefoxReality/issues/2230
+  data_reviews:
+    - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+  notification_emails:
+    - dmu@mozilla.com
```

Comment:
Does Firefox REality have a team mailing list or so? In this case, you should probably add it both here and to the metrics.

Reply:
Interesting! I was adding our team mailing list when doing the first stage migrating. IIRC, you would like me to use my personal mail? If you prefer to use to the team mail list, I think I can just put the team's one here because that group already include me.